src/main/java/com/github/splendor_mobile_game/websocket/handlers/exceptions/GameNotStartedException.java

```diff
@@ -0,0 +1,20 @@
+package com.github.splendor_mobile_game.websocket.handlers.exceptions;
+
+public class GameNotStartedException extends Exception {
```

Comment:
Add one line of java doc to this exception class. It can state the same what the name of class says, but we need to do these formalities, cause project needs to be documented.